packages/shared/lib/core/wallet/types/new-transaction-details.type.ts

```diff
@@ -17,13 +17,14 @@ type NewBaseTransactionDetails = {
     disableToggleGift?: boolean
     disableChangeExpiration?: boolean
     disableAssetSelection?: boolean
+    tokenMetadata?: TokenMetadata
```

Comment:
lets not add `tokenMetadata` as its not compatible with NFTs, you already have the metadata inside asset if the type is token